src/commands/info/about.ts

```diff
@@ -25,24 +25,58 @@ const execute = async (interaction: ChatInputCommandInteraction) => {
     .setTitle('Reimagine your Community')
     .addFields(
       {
-        name: 'Top-Tier Leveling Solution',
-        value:
-          'Elevate your Community to the next Level with Top-Tier Leveling, endless Customisability and more.',
+        name: t('fields.summary_title', {
```

Comment:
Where are these localisations supposed to come from? You gotta define them in the `/locales/{{lng}}/{{namespace}}.json` files.